R/liftover.R

```diff
@@ -23,9 +22,9 @@
 #'
 #' @return Data frame containing original bedpe data with new coordinates.
 #'
-#' @rawNamespace import(S4Vectors, except = c("merge", "second", "first", "union", "intersect", "setdiff", "setequal", "rename", "expand"))
+#' @rawNamespace import(S4Vectors, except = c("merge", "second", "first", "union", "intersect", "setdiff", "setequal", "rename", "expand", "head", "tail", "end", "start"))
 #' @rawNamespace import(GenomicRanges, except = c("start", "end", "merge", "shift", "union", "intersect", "setdiff", "reduce", "trim"))
-#' @import dplyr tidyr readr rtracklayer GAMBLR.data
+#' @import dplyr tidyr readr GAMBLR.data
```

Comment:
Just have a small question on why was the rtracklayer dropped here? But otherwise looks good!

Reply:
It seems all rtracklayer functions used in `liftover` are prefixes. 

When I ran the `liftover` after the changes, it worked. So I think this confirms that all of them are prefixed.

Reply:
It does not matter that much if it works or does not work when the rtracklayer functions are prefixed inside the liftover. Listing dependencies under `@input` has more value than just making the function work. Please remember that there are scenarios where the environment is empty and R is installed for the first time. We would ideally want the user to just install this package and it will auto-install all dependencies. Since you also removed the dependencies from DESCRIPTION and NAMESPACE the installation will go through but the `liftover` won't work since the dependencies were not properly listed and auto-installed. For you it worked because you already have rtracklayer installed in your R. Please revert this so GAMBLR.utils can properly deal with dependecies.

Reply:
`rtracklayer` wasn't removed from DESCRIPTION. On the contrary, I added it there.

Reply:
But it got removed from NAMESPACE since the package was removed from the import field of liftover.R

Reply:
Right - sorry, I meant the NAMESPACE. Since that one is auto-generated removing from @imports also dropped it from NAMESPACE

Reply:
done